.github/workflows/publish-edge.yml

```diff
@@ -0,0 +1,27 @@
+name: publish-edge
+on: workflow_dispatch
+permissions:
+  contents: read
+jobs:
+  upload-on-webstore:
+    runs-on: ubuntu-latest
+    environment: cd
+    outputs:
+      result: ${{ steps.webStorePublish.outcome }}
+    permissions:
+      actions: write
+    steps:
+      - uses: robinraju/release-downloader@a96f54c1b5f5e09e47d9504526e96febd949d4c2 # pin@v1.11
+        with:
+          tag: ${{ github.ref_name }}
+          fileName: "*"
+
+      - name: Publish on Microsoft Edge Addons
+        uses: wdzeng/edge-addon@v1
+        id: webStorePublish
+        with:
+          product-id: 18e6c4cd-6383-4f38-95e9-92a629f60817
```

Comment:
https://github.com/themoeway/yomitan/blob/f78e767b97ad71f27ebd406490ca82e792aaa456/.github/workflows/publish-chrome.yml#L89-L96

In the other action it seems the extension IDs are stored as secrets if we want to be consistent about it. (Though why use secrets and not variables since IDs aren't secret... 🤔)